### ISSUE TITLE
SurfaceView glitch: black line

### ISSUE THREAD

**Issue by droidpl** [dm77/barcodescanner#86]
The view appears to have a glitch that makes this widget unusable. I am testing against SDK 22 and the error is in 1.7 and 1.7.1. I am attaching a screenshot with the appearance I get from the widget.

I have configured it to read PDF_417 like this:

```
List<BarcodeFormat> formatList = new ArrayList<>(1);
formatList.add(BarcodeFormat.PDF_417);
mScannerView.setFormats(formatList);
```

If I can give you more feedback please, let me know.
Thanks.
![screen shot 2015-05-22 at 11 28 27](https://cloud.githubusercontent.com/assets/2143931/7767801/b70210fc-0075-11e5-9410-eee37552ffff.png)

**Comment by dm77** [comment]
Hi @feantury - Could this because you have other views in your layout that are taking up the available screen space? Could you please take a look at the sample XML to overlay elements that I provided in this issue #67 . Thanks.

**Comment by dm77** [comment]
Should be fixed in 1.9
